deploy/etc/docker/operator.Dockerfile

```diff
@@ -20,8 +20,9 @@
 # THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 FROM fwnetworking/python_base:latest
-COPY . /var/zeta/
-RUN pip3 install /var/zeta/deploy/
-RUN ln -snf /var/zeta/build/bin /trn_bin
+COPY build/operator /opt/zeta/operator
```

Comment:
I believe this should be 
`COPY ./src/mgmt/operator /opt/zeta/operator`
Since we don't actually build any binaries

Reply:
Now all deployments are based off build folder, not reference src